Header header
float64 outer_angle
float64 inner_angle
float64 thrust